<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Information_course</class>
 <widget class="QDialog" name="Information_course">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>课程信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;宋体&quot;;</string>
  </property>
  <widget class="QLabel" name="label_titile">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;添加/编辑课程信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>521</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_course_id">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;宋体&quot;;</string>
      </property>
      <property name="text">
       <string>课程编号</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_course_id</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_course_id"/>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineEdit_course_name"/>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="lineEdit_course_credit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_course_num">
      <property name="text">
       <string>课程人数</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_course_num</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_course_credit">
      <property name="text">
       <string>课程学分</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_course_credit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_course_capicity">
      <property name="text">
       <string>课程容量</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_course_capicity</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_course_type">
      <property name="text">
       <string>课程类型</string>
      </property>
      <property name="buddy">
       <cstring>comboBox_type</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_course_num"/>
    </item>
    <item row="1" column="5">
     <widget class="QComboBox" name="comboBox_type">
      <item>
       <property name="text">
        <string>必修</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>选修</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_course_capicity"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_course_name">
      <property name="text">
       <string>课程名称</string>
      </property>
      <property name="buddy">
       <cstring>lineEdit_course_name</cstring>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>400</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="confirm_btn">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="listView_student_n">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>171</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView_student_y">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>171</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_student">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择学生</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_teacher">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择教师</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_start_course">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>280</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选修课开启选课</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_teacher">
   <property name="geometry">
    <rect>
     <x>538</x>
     <y>200</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>41</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="select_student_btn">
      <property name="text">
       <string>&lt;-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_student_btn">
      <property name="text">
       <string>-&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
